### PR TITLE
[React-native]: Context breadcrumbs

### PR DESCRIPTION
**Pull request by brunokiafuka**
This PR adds the dynamic breadcrumbs to the side menu and page footer on RN.

### CODE REVIEW

File: src/components/RecentlyEditedBreadcrumbs.native.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+const mapStateToProps = (state: State, props: Omit<ContextBreadcrumbProps, 'simplePath'> & { path: Path }) => ({
+  simplePath: simplifyPath(state, props.path),
+})
+
+/**
+ * Varaint of ContextBreadcrumbs for recently edited with collapsing overflow.
+ */
+const RecentlyEditedBreadcrumbs = connect(mapStateToProps)(
```
Comment (Foosballfan) [inline src/components/RecentlyEditedBreadcrumbs.native.tsx:22]: shouldnt we be moving to useselector?

File: src/components/Superscript.native.tsx [hunk 2]
```diff
@@ -65,13 +66,12 @@ const Superscript: FC<SuperscriptProps> = ({ empty, numContexts, showSingle, sup
   //   .reduce((charCount, child) => charCount + child.length, 0)
 
   return (
-    <Text style={commonStyles.whiteText}>
+    <Text>
       {
         !empty && superscript && numContexts! > (showSingle ? 0 : 1) ? (
           <Text>
-            {' '}
             {/* Make the container position:relative so that the modal is positioned correctly */}
```
Comment (Foosballfan) [inline src/components/Superscript.native.tsx:73]: does this comment apply?

File: src/components/Superscript.native.tsx [hunk 3]
```diff
@@ -65,13 +66,12 @@ const Superscript: FC<SuperscriptProps> = ({ empty, numContexts, showSingle, sup
   //   .reduce((charCount, child) => charCount + child.length, 0)
```
Comment (Foosballfan) [inline src/components/Superscript.native.tsx:66]: are we planning to use this commented out code?

File: src/components/RecentlyEditedBreadcrumbs.native.tsx (hunk 1 above)
Comment (brunokiafuka) [reply src/components/RecentlyEditedBreadcrumbs.native.tsx:22]: oh yeah, will fix it now.

File: src/components/Link.native.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+// import { EM_TOKEN } from '../constants'
+import { scrollCursorIntoView, search, searchContexts, setCursor, toggleSidebar } from '../action-creators'
+import {
+  clearSelection,
+  // decodeCharacterEntities,
```
Comment (Foosballfan) [inline src/components/Link.native.tsx:7]: do we want all these comments?

File: src/components/Superscript.native.tsx (hunk 3 above)
Comment (brunokiafuka) [reply src/components/Superscript.native.tsx:66]: removed

File: src/components/Superscript.native.tsx (hunk 2 above)
Comment (brunokiafuka) [reply src/components/Superscript.native.tsx:73]: removed

**Review by Foosballfan — APPROVED** [review]

**Review by shresthabijay — APPROVED** [review]
Looks good!

**Pull request merged**